src/pathfinding_service/model/channel_view.py

```diff
@@ -42,25 +42,22 @@ def __post_init__(self) -> None:
         if self.capacity is None:
             self.capacity = self.deposit
 
-    # TODO: define another function update_deposit
+    def update_deposit(self, deposit: TokenAmount = TokenAmount(0)) -> None:
+
+        self.capacity = TokenAmount(self.capacity + deposit - self.deposit)
+        self.deposit = TokenAmount(self.deposit + deposit)
```

Comment:
This is wrong, `deposit` is a total value.

Reply:
ok I give you that